src/env-inl.h

```diff
@@ -512,6 +513,11 @@ inline void Environment::set_fs_stats_field_array(double* fields) {
   fs_stats_field_array_ = fields;
 }
 
+inline AliasedBuffer<uint32_t, v8::Uint32Array>&
+Environment::scheduled_immediate_count() {
```

Comment:
I'm not really a fan of returning a mutable reference.  I realize other AliasedBuffer things do that too but IMO they shouldn't, it obscures whether call sites deal with a copy or a reference.

(I'll allow that it looks like a little nicer coupled with array syntax.)

Reply:
I know. :) I don’t like it when one can’t tell the difference either, but in these cases it should be pretty clear.